docs/installation/windows.md

```diff
@@ -1,161 +1,263 @@
+cmake --build . --config Release --target install
+```
+
+![](../img/windows-command-line-success.png)
+
+Add the following to the Windows PYTHONPATH environment variable or run the following in the command line.
+
+```bash
+set PYTHONPATH=C:\local\helics-v1.0.2\python;%PYTHONPATH%
+```
+
+If you open a interactive Python session and import helics, you should be able to get the version of `helics` and an output that is similar to the following.
+
+```bash
+$ ipython
+Python 3.6.4 |Anaconda, Inc.| (default, Jan 16 2018, 12:04:33)
+Type 'copyright', 'credits' or 'license' for more information
+IPython 6.2.1 -- An enhanced Interactive Python. Type '?' for help.
+
+In [1]: import helics
+
+In [2]: helics.helicsGetVersion()
+Out[2]: '1.0.2 (04-28-18)'
+
+```
+
+![](../img/windows-python-success.png)
+
+## Building GridLAB-D and HELICS From Source on Windows with MSYS2 ##
+### Overview ###
+The latests release of GridLAB-D which is 4.1 will automatically come with the HELICS library bundled with it. However, The 4.1 release is not stable yet. Until then GridLAB-D and HELICS can be compiled together from source. The latest Develop branch of GridLAB-D can only be built on Windows using MSYS2. So both GridLAB-D and HELICS must be compiled using MSYS2. This page will layout the setting up of MSYS2 to compile and install HELICS and GridLAB-D. This guide will describe all the packages and install instructions for a 64bit build.
+
+### Setting up MSYS2 ###
+MSYS2 provides a linux like terminal environment on your Windows system. MSYS2 can be installed from [here](https://www.msys2.org/). Once MSYS2 has been installed start up msys2.exe. Follow first time updates as described on the MSYS2 website. After MSYS2 has been successfully updated Some packages need to be installed in order to configure and build GridLAB-D and HELICS. The following packages need to be installed:
+- base-devel
+- mingw-w64-x86_64-toolchain
+- git
+- mingw-w64-x86_64-cmake
+- mingw-w64-x86_64-boost
+- mingw-w64-x86_64-qt5 (only if you want to be able to run cmake-gui which this guide recommends.)
+- mingw-w64-x86_64-dlfcn
+- mingw-w64-x86_64-xerces-c
+- mingw-w64-x86_64-zeromq
+
+All packages can be installed by typing the following:
+```bash
+$ pacman -S --needed base-devel mingw-w64_x86_64-toolchain mingw-w64-x86_64-cmake mingw-w64-x86_64-boost mingw-w64-x86_64-qt5 mingw-w64-x86_64-dlfcn mingw-w64-x86_64-xerces-c
+```
+One issue that the boost package has when building with GridLAB-D with HELICS on MSYS2 is all the boost libraries have a -mt in their names. In order for GridLAB-D to find these libraries the -mt must be removed from the libboost*-mt.dll files located in /mingw64/bin. Work is in progress to remove this modification in MSYS2 builds.
+
+After all this has been done /mingw64/bin must be added to the PATH environment variable.
+```bash
+$ export PATH=$PATH:/mingw65/bin

```

Comment:
should be ming64 and this should be in the system path already, so I am very unclear why this is necessary.  That is one of the system directories and is part of the default path.  Everything in that directory is available from the command line directly and is easily found.

Reply:
it is only if people started up the MSYS2 MinGW 64-bit terminal. If they just start the MSYS2 MSYS terminal it is not and you have to add it. This is just me trying to cover bases.